core/src/main/resources/jenkins/model/Jenkins/manage.jelly

```diff
@@ -47,11 +47,13 @@ THE SOFTWARE.
     <st:include page="downgrade.jelly" />
     </div>
 
-    <j:forEach var="m" items="${it.managementLinks}">
-      <l:hasPermission permission="${m.requiredPermission}">
+    <j:forEach var="category" items="${it.categorizedManagementLinks.entrySet()}">
+      <h2>${category.key.label}</h2>
```

Comment:
Just to confirm, I'm assuming there can be no instance of showing a category without entries right? Is that why an emptiness check is not needed?

Reply:
Yes, the way I'm populating the map I'm iterating over prevents empty categories (which results in a bit of duplication when it comes to determining item visibility).